n3fit/src/n3fit/layers/observable.py

```diff
@@ -15,6 +17,44 @@ def is_unique(list_of_arrays):
     return True
 
 
+def compute_pdf_boundary(pdf, q0_value, xgrid, n_std, n_replicas):
+    """
+    Computes the Unpolarized PDF set to be used as a Boundary Condition and
+    convert it into a Tensor object that can be understood by the convolution.
+
+    Parameters
+    ----------
+    pdf: validphys.core.PDF
+        a validphys instance of the Unpolarized PDF set
```

Comment:
```suggestion
        a validphys PDF instance to be used as a boundary PDF set
```

I think this same function (maybe a generalized version) will be useful also going forward for the final universal qcd fit well beyond the polarized case